.github/workflows/pr-open.yml

```
@@ -2,49 +1,115 @@
+      - name: Upload report
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: report
+          path: apps/web/playwright-report/
+
+      - name: Save PR number
+        if: always()
+        env:
+          PR_NUMBER: ${{ github.event.number }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:6: Double quote to prevent globbing and word splitting

(shellcheck)